src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { useTodos } from '../utils/TodosContext';
+import { TodoFilter } from './TodoFilter';
+
+export const TodoFooter: FC = () => {
+  const { todos, removeTodo } = useTodos();
+  const isClearButtonActive = todos.some(todo => todo.completed);
+  const countActive = todos.filter(todo => !todo.completed).length;
```

Comment:
```suggestion
  const activeTodosCount = todos.filter(todo => !todo.completed).length;
```
not critical but will be more informative